docs/application/vscode-ext/web.md

```diff
@@ -79,232 +51,237 @@ To download emulator images, you can use Tizen Package Manager or Tizen Emulator
   ![Run Your Tizen App](media/tizen_web15.png)
 
-> **Note**  
->
-> If you have a Tizen device and you want to deploy the application to it, you must first generate, and then add a certificate.
->
-> In addition, vscode terminal displays successful launch log messages:
-> 
-> 1. debug 0 means normal launch.
-> 2. debug 1 means launching an Inspector available.
+        > **Note**  
+        >
+        > If you have a Tizen device and want to deploy the application into it, then you must first generate a certificate and add it.
+        >
+        > In addition, `vscode` terminal displays successful launch log messages:
+
+      - debug `0` means normal launch.
+      - debug `1` means launching an inspector available from command line.
 
 ### Debug Application in Emulator
 
-To debug application, follow these steps:
+1. In VS Code, click **View** > **Command Palette**.
+2. In the input field of the **Command Palette** that appears, type **Debug Tizen App** and press **Enter**.
+3. Enter the name of the Tizen Web App you want to run. 
+
+    A successful notification appears and the app is launched. Also, you will see a successful launch log in the terminal on the vscode.
 
-> **Note**
->
-> Ensure the following:
-> - Emulator is launched
-> - Device is connected to the PC
-> - Chrome browser is installed
+    - debug `0` means normal launch.
+    - debug `1` means launching an inspector available from command line.
 
-1. Open **Command Palette** and enter **Debug Tizen App**.
-2. Enter the name of the Tizen Web App you want to run.
-3. A successful popup is created and the app is launched.
+    When port information is displayed in **Terminal**, it means that it is launched in debug mode.
 
-> **Note**
-> 
-> In addition, `vscode` terminal displays successful launch log messages:
->
-> - debug `0` means normal launch.
-> - debug `1` means launching an inspector available from command line
-> - additionally, debug the connected port as shown in the following log
+    ![Debugging you application](media/tizen_web16.png)
 
-  ![Debugging you application](media/tizen_web16.png)
+    > **Note**
+    >
+    > Ensure the following:
+    > - Emulator is launched, or device is connected to the PC
+    > - Chrome browser is installed
 
 4. Connect **Debug Tizen App** with **Inspector**.
-5. Open **Command Palette** and enter **Run Inspector**.
-6. Enter the port number on your local PC. For example, 9999
+5. Open **Command Palette** and type **Run Inspector**.
+6. Enter the port number to be used on the local computer. For example, 9999.
 7. In Chrome browser, the **Inspector** automatically launches and associates with the launched app.
+![Debugging you application](media/tizen_web17.png)
 
-    ![Debugging you application](media/tizen_web17.png)
+    > **Note** 
+    > 
+    > If you have a Tizen device and you want to deploy the application to it, then you must first generate a certificate and then add the certificate.
```

Comment:
Please change line 229 to:

If you have a Tizen device and want to deploy the application into it, then you must first generate a certificate and add it.

Reply:
done.